src/Form/Type/FormatterType.php

```diff
@@ -156,11 +132,11 @@ public function buildView(FormView $view, FormInterface $form, array $options):
 
         $view->vars['format_field_options'] = $options['format_field_options'];
 
-        $defaultConfig = $this->configManager->getDefaultConfig();
+        $defaultConfig = $this->ckEditorConfiguration->getDefaultConfig();
 
-        if ($this->configManager->hasConfig($defaultConfig)) {
-            $ckeditorConfiguration = $this->configManager->getConfig($defaultConfig);
-        } else {
+        try {
+            $ckeditorConfiguration = $this->ckEditorConfiguration->getConfig($defaultConfig);
+        } catch (Exception $e) {
```

Comment:
Could we trust a more specific exception?

Reply:
Like `FOSCKEditorException` ?
You want me to throw an exception instead of sending an empty array ?
Like
```
throw new \RuntimeException('Unable to find CKEditor default config, check your fos_ck_editor.yaml file in your config folder, error is: '.$e->getMessage(), $e->getCode(), $e);
```

Reply:
I don't know what exceptions are able to be thrown from `getConfig()` method, but I think the exception catched here must be narrow enough in order to avoid silencing other unrelated potential exceptions.

Reply:
Hey guys, hope this hasn't gone stale, just my thoughts - the getConfig() method throws a ConfigException::configDoesNotExist exception when the config requested is not set. I don't see why you would want to catch that exception and ignore it by setting the the editor `$ckeditorConfiguration` to an empty array. I think it would be best to just let the FOSCKEditorBundle handle throwing the exception if an expect config does not exist (since that indicates an issue with the bundle configuration).

Reply:
@timwentzell Thanks for the tip
You're right, it's better to let FOSCkEditorBundle handle the error.
I will correct this, and also look at the @wbloszyk comment below